File: Java/Dipendendi_Scuola/Dipendente.java
public abstract class Dipendente {
    protected String nominativo;
    protected String sesso;
    protected java.time.LocalDate data_di_nascita;
    protected double stipendio_base_mensile;

    public Dipendente(String nominativo, String sesso, java.time.LocalDate data_di_nascita, double stipendio_base_mensile) {
        this.nominativo = nominativo;
        this.sesso = sesso;
        this.data_di_nascita = data_di_nascita;
        this.stipendio_base_mensile = stipendio_base_mensile;
    }
    // Getters
    public String getNominativo() {
        return nominativo;
    }
    public String getSesso() {
        return sesso;
    }
    public java.time.LocalDate getData_di_nascita() {
        return data_di_nascita;
    }
    public double getStipendio_base_mensile() {
        return stipendio_base_mensile;
    }

    // Setters
    public void setNominativo(String nominativo) {
        this.nominativo = nominativo;
    }
    public void setSesso(String sesso) {
        this.sesso = sesso;
    }
    public void setData_di_nascita(java.time.LocalDate data_di_nascita) {
        this.data_di_nascita = data_di_nascita;
    }
    public void setStipendio_base_mensile(double stipendio_base_mensile) {
        this.stipendio_base_mensile = stipendio_base_mensile;
    }

    // Method
    public abstract String toString();

    /**
     * Calcola lo stipendio annuale del dipendente
     * @return double stipendio annuale
     */
    public double calcolaStipendio() {
        return stipendio_base_mensile * 12;
    }
}

File: Flutter/Dart1/Dipendente.dart
class Dipendente {
  String _nominativo;
  String _sesso;
  String _dataDiNascita;
  double _stipendioBase;

  //Costruttore con parametri nominali
  Dipendente({String nominativo = "", String sesso = "", String dataDiNascita = "00-00-0000", double stipendioBase = 0.0})
  : _nominativo = nominativo,
    _sesso = sesso,
    _dataDiNascita = dataDiNascita,
    _stipendioBase = stipendioBase;

  //Getters
  String get nominativo => _nominativo;
  String get sesso => _sesso;
  String get dataDiNascita => _dataDiNascita;
  double get stipendioBase => _stipendioBase;

  //Metodo
  String toString() {
    return "Dipendente: ${_nominativo}, Sesso: ${_sesso}, Data di Nascita: ${_dataDiNascita}, Stipendio Base: ${_stipendioBase}";
  }
}